apps/frontend/app/(main)/dashboard/[websiteId]/page.tsx

```
@@ -85,62 +95,65 @@ function StatusCircle({ status }: { status: UptimeStatus }) {
 
+    // Function to calculate time ago for tooltip based on time range
+    const getTimeAgo = (index: number) => {
+        switch (timeRange) {
+            case "1w":
+                return `${7 - index} day${index !== 6 ? "s" : ""} ago`;
+            case "1m":
+                return `${30 - index} day${index !== 29 ? "s" : ""} ago`;
+            case "1y":
+                return `${12 - index} month${index !== 11 ? "s" : ""} ago`;
+            default: // 30m
+                const minutesAgo = 30 - index * 3;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)